tests/foreman/__init__.py

```diff
@@ -0,0 +1,3 @@
+from robottelo.config import settings
+
+settings.configure()
```

Comment:
Do the tests themselves require that settings to be read and parsed just once? Let's imagine for a moment that reading and parsing the configuration file has zero cost in CPU cycles and time. In that case, would it be acceptable for every single bit of code that needs access the settings to parse the configuration file again?